src/main/java/domain/Pojos/PojoRepList.java

```diff
@@ -0,0 +1,67 @@
+package domain.Pojos;
+
+import java.util.List;
+
+public class PojoRepList {
+    private String state;
+    private List<String> name;
+    private List<String> district;
+    private List<String> party;
+    private List<String> officeRoom;
+    private List<String> phone;
+    private List<String> committee_assignment;
+
```

Comment:
which approach is better do you think?
each rep row(line) in a given state as an object? or this way